src/main/drivers/barometer/barometer_lps.c

```diff
@@ -0,0 +1,166 @@
+#define LPS25HB_REF_P_XL    0x08
+#define LPS25HB_REF_P_L     0x09
+#define LPS25HB_REF_P_H     0x0A
+#define LPS25HB_WHO_AM_I    0x0F
+#define LPS25HB_RES_CONF    0x10
+#define LPS25HB_CTRL1       0x20
+#define LPS25HB_CTRL2       0x21
+#define LPS25HB_CTRL3       0x22
+#define LPS25HB_CTRL4       0x23
+#define LPS25HB_INT_CFG     0x24
+#define LPS25HB_INT_SOURCE  0x25
+#define LPS25HB_STATUS      0x27
+#define LPS25HB_OUT_XL      0x28
+#define LPS25HB_OUT_L       0x29
+#define LPS25HB_OUT_H       0x2A
+#define LPS25HB_TEMP_OUT_L  0x2B
+#define LPS25HB_TEMP_OUT_H  0x2C
+#define LPS25HB_FIFO_CTRL   0x2E
+#define LPS25HB_FIFO_STATUS 0x2F
+#define LPS25HB_THS_PL      0x30
+#define LPS25HB_THS_PH      0x31
+#define LPS25HB_RPDS_L      0x39
+#define LPS25HB_RPDS_H      0x3A
+//=======================================Constants=============================================// 
+#define LPS25HB_ID          0xBD
+#define LPS25HB_READ        0x80
+#define LPS25HB_MULT        0x40
+//======================================ODR Values=============================================//
+// CTRL1 value
+#define LPS25HB_ODR_ONE_SHOT        0x00
+#define LPS25HB_ODR_1               0x10
+#define LPS25HB_ODR_7               0x20
+#define LPS25HB_ODR_12_5            0x30
+#define LPS25HB_ODR_25              0x40
+//======================================Average number=============================================//
+#define LPS25HB_AVP_8               0x00
+#define LPS25HB_AVP_32              0x01
+#define LPS25HB_AVP_128             0x02
+#define LPS25HB_AVP_512             0x03
+#define LPS25HB_AVT_8               0x00
+#define LPS25HB_AVT_16              0x04
+#define LPS25HB_AVT_32              0x08
+#define LPS25HB_AVT_64              0x0C
+//======================================Moving average number=============================================//
+#define LPS25HB_NO_MA               0x00
+#define LPS25HB_MA_2                0x01
+#define LPS25HB_MA_4                0x03
+#define LPS25HB_MA_8                0x07
+#define LPS25HB_MA_16               0x0F
+#define LPS25HB_MA_32               0x1F
+
+bool lpsWriteCommand(busDevice_t *busdev, uint8_t cmd, uint8_t byte) {
+    return spiBusWriteRegister(busdev, cmd, byte);
+}
+
+bool lpsReadCommand(busDevice_t *busdev, uint8_t cmd, uint8_t len, uint8_t *data) {
+    return spiBusReadRegisterBuffer(busdev, cmd | 0x80 | 0x40, data, len);
+}
+
+static void lpsOn(busDevice_t *busdev, uint8_t CTRL1_val) {
+    lpsWriteCommand(busdev, LPS25HB_CTRL1, CTRL1_val | 0x80);
+    delay(500);
+}
+
+static void lpsOff(busDevice_t *busdev) {
+    lpsWriteCommand(busdev, LPS25HB_CTRL1, 0x00 | (0x01 << 2));
+}
+
+uint32_t rawP = 0;
+uint32_t rawT = 0;
+
+static void lps_nothing(baroDev_t *baro) {
+	UNUSED(baro);
+	return;
+}
+
+static void lps_read(baroDev_t *baro) {
+    uint8_t status = 0x00;
+    lpsReadCommand(&baro->busdev, LPS25HB_STATUS, 1, &status);
+    if (status & 0x03) {
```

Comment:
Shouldn't this wait for both temperature AND pressure data?
`if ((status & 0x03) == 0x03) {`

Reply:
If we wanna read both then yes, but the question is if we want to read temperature

Reply:
OK, but it should be commented in code (I assume that old reading will be used)